packages/app/src/components/profile/StreamButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from 'react';
```

Comment:
This actually isn't a stream button. This will open a following list on the sidebar. Think discord-esque